### ISSUE TITLE
Crab struggles to grab tie from NPC

### ISSUE THREAD

**Issue by FoodAbuse** [FoodAbuse/stabo-crabo-game#21]
**Description**
The tie on the first puzzle NPC is proving difficult to grab in my tests which I fear may confuse players into thinking that grabbing the tie isn't part of the puzzle. I have attempted to fix it but I don't want to break the code. I believe it is because the collider manager script is detecting too many colliders that it can interact with and can't decide on which to attempt to grab, but in attempting to fix this I found no solution. Hoping that you may save us with your coding wisdom!

**Version**
V0.2-Alpha (Semester 2, September Pre PGF Build)

**To Reproduce**
Attempt to grab tie from sleeping NPC

**Expected Behavior**
Ideally, the tie would be outlined and be easy to grab from a distance instead of having to spam click and pixel hunt 

**Screenshots**
_N/A_

**Desktop System**
OS: Windows 11

**Comment by ZachariaWinter** [comment]
Agreed,
I can look at creating a 'prioty order' for colliders.
that when multiple colliders are detected, the priority one is the one tyhat is highlighted and interacted with.
Currently it just highlights the first collider in the list

**Comment by ZachariaWinter** [comment]
I have changed the tie collider from a mesh collider, to a larger than the model capsule collider.
+ easier to grab
- tie rolls around a bit on the floor now
- when held, the tie is not grabbed from further away and floats a little in your hand
   > I don't believe this negatively affects the experience more than the tie being hard to grab

I have also created a priority system in the collider collection scripts, which so far seems to work.
Tie is at priority 5.
Everything else at time of writing is at prioprity 0 (the default)
